CHANGELOG.md

```diff
@@ -11,34 +11,13 @@ Full documentation for rocFFT is available at [rocfft.readthedocs.io](https://ro
   is no longer performed by each node. We designed a generic algorithm to
   test and pick the best assignment path.
   - With the help of FuseShim, we can achieve more kernel-fusions as possible.
-
+- Split 2D device code into separate libraries.
```

Comment:
This line is obsoleted by the one at the start of the 1.0.15 changelog - now our device library is split into 4 equal libraries.  There is no separate split for 2D code.